src/commands/create/index.ts

```diff
@@ -107,7 +107,7 @@ export const handler = async (predefinedFolderName: string | undefined, options:
           ],
           required: true,
         },
-      ]);
+      ], {projectType: options.project });
```

Comment:
Is it expected change?

Reply:
That shouldn't be a part of this PR, I added by accident when testing zksync-cli. We have a `--project` option but it it's ignored and the CLI still asks to choose a project type.